src/main/java/me/xemor/yaggaskabble/LeaderboardCommand.java

```diff
@@ -31,14 +31,25 @@ public void onSlashCommandInteraction(SlashCommandInteractionEvent event) {
             event.reply("You have entered an invalid winning alignment!").queue();
             return;
         }
-
+        String leaderboardString;
         List<Player> players = yaggaskabble.getPlayers();
-        players.sort(Comparator.comparingDouble(player -> player.getSkillRatingForAlignment(alignment).conservativeRating()));
-        Collections.reverse(players);
-
-        String leaderboardString = players.stream()
+        
+        if (alignment == Alignment.BOTH) {
```

Comment:
I was thinking you'd pull it out into its own separate command originally, but if you still want to be able to do /leaderboard both, probably just make it handle the alignment argument in a way that doesn't do Alignment.valueOf().